CoreScriptsRoot/ServerCoreScripts/ClientChat/ChatChannel.lua

```diff
@@ -0,0 +1,163 @@
+local source = [[
+local module = {}
+--////////////////////////////// Include
+--//////////////////////////////////////
+local modulesFolder = script.Parent
+local moduleTransparencyTweener = require(modulesFolder:WaitForChild("TransparencyTweener"))
```

Comment:
is it worth it having an entire module devoted to just transparency tweening? I think @SolarCrane has made a few tweening libraries that are pretty generic